lib/command/src/Obelisk/Command/Deploy.hs

```diff
@@ -258,7 +261,8 @@ nix-env -p /nix/var/nix/profiles/system --set "${bashEscape outPath}"
 /nix/var/nix/profiles/system/bin/switch-to-configuration boot
 booted="$(readlink /run/booted-system/{initrd,kernel,kernel-modules})"
 built="$(readlink /nix/var/nix/profiles/system/{initrd,kernel,kernel-modules})"
-if [ "$booted" = "$built" ]; then
+echo "${bashEscape (show reboot)}"
+if [[ "$booted" = "$built" && "${bashEscape (show reboot)}" == "${bashEscape (show False)}" ]]; then
```

Comment:
It seems to me like it might be better to just template in the reboot if the flag is set rather than doing logic in bash